plugins/modules/content_create_template.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
```

Comment:
I think module name should be content_template and handle create, delete and update as well